Blockzilla/SettingsViewController.swift

```diff
@@ -188,26 +190,42 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         let usageDataSubtitle = String(format: UIConstants.strings.detailTextSendUsageData, AppInfo.productName)
         let usageDataToggle = BlockerToggle(label: UIConstants.strings.labelSendAnonymousUsageData, setting: SettingsToggle.sendAnonymousUsageData, subtitle: usageDataSubtitle)
         let safariToggle = BlockerToggle(label: UIConstants.strings.toggleSafari, setting: SettingsToggle.safari)
+        let homeScreenTipsToggle = BlockerToggle(label: UIConstants.strings.toggleHomeScreenTips, setting: SettingsToggle.showHomeScreenTips)
+        
         var toggles = [Int : BlockerToggle]()
         if let biometricToggle = createBiometricLoginToggleIfAvailable() {
             toggles = [
                 1: blockFontsToggle,
                 2: biometricToggle,
                 3: usageDataToggle,
-                6: safariToggle
+                6: safariToggle,
+                7: homeScreenTipsToggle
             ]
-        }
-        else {
+            
+            if !UserDefaults.standard.bool(forKey: BrowserViewController.userDefaultsShareTrackerStatsKeyNEW) {
+                toggles[7] = nil
+            }
+        } else {
             toggles = [
                 1: blockFontsToggle,
                 2: usageDataToggle,
-                5: safariToggle
+                5: safariToggle,
+                6: homeScreenTipsToggle
             ]
+            
+            if !UserDefaults.standard.bool(forKey: BrowserViewController.userDefaultsShareTrackerStatsKeyNEW) {
```

Comment:
I would consider having one check to add in the homeScreenTipsToggle. You can check the default and then if it's true, you can add the safariToggle index + 1 with the homeScreenTipsToggle.